App.js

```diff
@@ -1,129 +1,148 @@
-import { StatusBar } from 'expo-status-bar';
-import { FlatList, StyleSheet, Text, View } from 'react-native';
-// metro doesn't support package.json exports field @10up/headless-core/react works with web bundlers.
-import { BlocksRenderer } from '@10up/headless-core/dist/cjs/react/components/BlocksRenderer';
-import { isBlockByName } from '@10up/headless-core';
+import headlessConfig from "./headless.config";
 
-const mockWordPressPostContentResponse = `
-<p data-wp-block-name='core/paragraph'>this is a paragraph</p>
-<ul data-wp-block-name='core/list'>
-<li data-wp-block-name='core/list-item'>
-This is a list item 1
-</li>
-<li data-wp-block-name='core/list-item'>
-This is a list item 2
-</li>
-</ul>`;
+globalThis.__10up__HEADLESS_CONFIG = { ...headlessConfig };
```

Comment:
I'm not proud of this but this is needed for now. In next.js we inject the config at compile time. There might be away to this with metro and RN but I didn't have time to look into it.